seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/sink/file/ClickhouseFileSinkWriter.java

```diff
@@ -285,6 +285,7 @@ private List<String> generateClickhouseLocalFiles(String clickhouseLocalFileTmpF
                         + "\"");
         command.add("-N");
         command.add("\"" + "temp_table" + uuid + "\"");
+        command.add("-d _local");
```

Comment:
dose the old version support this parameter too?

Reply:
I tested the 2023-2024 version, not sure about earlier versions